README.md

```diff
@@ -75,6 +75,11 @@ Please refer to the following model zoo for more details.
 - [classification model zoo](docs/source/model_zoo_cls.md)
 - [detection model zoo](docs/source/model_zoo_detection.md)
 
+## Data Hub
+
+EasyCV summarized various datasets in different fields.
```

Comment:
EasyCV have collected dataset info for different senarios,  making it easy for users to fintune or evaluate models in EasyCV modelzoo

Reply:
done!